server/api/controllers/article.js

```diff
@@ -11,3 +15,133 @@ export const createArticle = async (req, res) => {
     return res.status(502).send({ status: 'Error', message: 'OOPS! an error occurred while trying to create your article, you do not seem to be logged in or signed up, log in and try again!' });
   }
 };
+
+/**
+* @export
+* @function likeArticle
+* @param {Object} req - request received
+* @param {Object} res - response object
+* @returns {Object} JSON object (JSend format)
+*/
+export const likeArticle = async (req, res) => {
+  try {
+    const articleId = req.params.id;
```

Comment:
@Ibidapo lines `28` and `29` can be changed to `const { params: { id: articleId }, user: { id : userId } } = req;`